src/main/twirl/com/lightbend/lagom/docs/documentation.scala.html

```diff
@@ -81,7 +81,7 @@
 
             };
 
-            var lagomFacetFilters = ['version:current'];
+            var lagomFacetFilters = ['version:@lang.name'];
```

Comment:
I wonder how long this bug existed for!

Reply:
How does this work?

Won't this try to search by `version:scala`  instead of `version:1.6.x`?

Long ago, when extending search to be cross-project, the agreement was that all searches would be on `current`  as a way to push users towards latest. So, hardcoding `version:current` was not a bug but a feature.

Reply:
`lang` is an instance of `LanguageVersion`, the `name` field of `LanguageVersion` contains the name of the version, not the name of the language.

That seems really odd to me re:hardcoding current... if I'm in the 1.5.x docs, because I'm using Lagom 1.5.x, then when I search, I want to get 1.5.x results. I don't see how returning current results helps anyone, that would frustrate me as a user. I don't think anyone ever says "we should upgrade to the latest version because when we search in the docs we get silently switched to the latest version in the docs", that just would never appear as a reason. If it's a reason for any decisions being made, it might be a reason why people might stop using Lagom... "the docs are really hard to use because when you search you get switched to the latest version".

Reply:
I'm sure the people that made that decision understand the trade-offs and still preferred `current`.

Let me rephrase: it's not a bug it's an opinion.